pkg/pool-stable/contracts/meta/OracleMiscData.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-solidity-utils/contracts/helpers/WordCodec.sol";
+
+/**
+ * @dev This module provides an interface to store different pieces of information used by pools with a price oracle.
```

Comment:
Should we take it out of the `meta` directory?

Reply:
I thought about placing it in the `pool-utils/oracle` dir but it is actually custom to each storage layout, and this one applies only to the stable pool for now. I think we can do the same learning process we did with the `2TokensWeightedPool` if we need it in the future